packages/devtools_app/lib/src/config_specific/drag_and_drop/_drag_and_drop_web.dart

```diff
@@ -49,7 +45,8 @@ class _DragAndDropWebState extends DragAndDropState {
   }
 
   void _onDragOver(MouseEvent event) {
-    super.dragOver();
+    super.dragOver(event.offset.x, event.offset.y);
```

Comment:
not a new issue but it is surprising to call super with a method other than the name if the current method. Can that be avoided?

Reply:
`super.dragOver()` doesn't take a MouseEvent because that is an html specific import. I removed `super` to make it more clear that this is not an override method